shell/platform/tizen/flutter_tizen_texture_registrar.cc

```diff
@@ -95,7 +95,16 @@ FlutterTizenTextureRegistrar::CreateExternalTexture(
           texture_info->pixel_buffer_config.user_data);
       break;
     case kFlutterDesktopGpuBufferTexture:
+      if (engine_->renderer()->IsSupportedExtention(
```

Comment:
Taking `ExternalTextureGLExtention_None` into account and removing code duplication:
```suggestion
      ExternalTextureGLExtention gl_extension = ExternalTextureGLExtention_None;
      if (engine_->renderer()->IsSupportedExtention(
              "EGL_TIZEN_image_native_surface")) {
        gl_extension =
            ExternalTextureGLExtention_EGL_TIZEN_image_native_surface;
      } else if (engine_->renderer()->IsSupportedExtention(
                     "EGL_EXT_image_dma_buf_import")) {
        gl_extension = ExternalTextureGLExtention_EGL_EXT_image_dma_buf_import;
      }
```